src/m365/planner/commands/task/task-get.ts

```diff
@@ -0,0 +1,61 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import commands from '../../commands';
+import { Task } from '../../Task';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  taskId: string;
+}
+
+class PlannerTaskGetCommand extends GraphItemsListCommand<Task> {
+  public get name(): string {
+    return commands.TASK_GET;
+  }
+
+  public get description(): string {
+    return 'Retrieve the details of the specified planner task';
+  }
+
+  public defaultProperties(): string[] | undefined {
```

Comment:
`defaultProperties` are meant to be used in `list` commands where we return multiple items. Since we return a single item here, we don't need it as it doesn't serve any purpose